snippets/css-snippets.cson

```diff
@@ -543,7 +549,7 @@
   'box-shadow':
     'prefix': 'bs'
     'body': """
-        // h-length, v-length, blur, spread, colour
```

Comment:
Use `//` for scss, sass, and less.  For css, use `/* */`.  This will mean we will have to break out the shadow snippets to a third section.